src/data/vue.js/primary-options.js

```diff
@@ -0,0 +1,22 @@
+const options = [
+  { value: "create", label: "create" },
+  { value: "ui", label: "ui" },
+  { value: "serve", label: "serve" },
+  { value: "build", label: "build" },
+  { value: "lint", label: "lint" },
+  { value: "add", label: "add" },
+  { value: "invoke", label: "invoke" },
+  { value: "inspect", label: "inspect" },
+  { value: "install", label: "install" },
+  { value: "upgrade", label: "upgrade" },
+];
+
+export const primaryOptions = options.sort((x, y) => {
+  if (x.value < y.value) {
+    return -1;
+  }
+  if (x.value > y.value) {
+    return 1;
+  }
+  return 0;
+});
```

Comment:
I see we're duplicating this in every folder containing data. Shall we create a new folder called utils and move this to it and then import? Can you do this change for other folders which contain data as well?

@icy-r